test/js/property-dictionary.js

```diff
@@ -0,0 +1,54 @@
+suite('PropertyDictionary', function() {
+  setup(function() {
+    propertyDictionary = getInstance();
+  });
+
+  test('The isSupportedProperty method should return true if the input is a valid CSS property', function() {
+    assert.isTrue(propertyDictionary.isSupportedProperty('height'));
+  });
+
+  test('The isSupportedProperty method should return false if the input is not a valid CSS property', function() {
+    assert.isFalse(propertyDictionary.isSupportedProperty('lemons'));
+  });
+
+  test('The _lengthValueHasPercentage method should return true if a CalcLength object has a' +
+      'value that is not null for its percent property', function() {
+    assert.isTrue(propertyDictionary._lengthValueHasPercentage(new CalcLength({percent: 10})));
+  });
+
+  test('The _lengthValueHasPercentage method should return true if a SimpleLength object has a type of percent', function() {
+    assert.isTrue(propertyDictionary._lengthValueHasPercentage(new SimpleLength(10, 'percent')));
+  });
+
+  test('The isValidInput method should return true if the property can accept the style value entered', function() {
+    assert.isTrue(propertyDictionary.isValidInput('height', new SimpleLength(9.2, 'px')));
+  });
+
+  test('Testing the isValidInput method with a non Length value property' , function() {
+    assert.isTrue(propertyDictionary.isValidInput('pitch-range', new NumberValue(5)));
```

Comment:
Just add this line to the above test - it's essentially testing the same thing as

`The isValidInput method should return true if the property can accept the style value entered`